libs/orders/src/lib/components/order-list/order-view-dialog.tsx

```diff
@@ -0,0 +1,263 @@
+import { addDecimalsFormatNumber } from '@vegaprotocol/utils';
+import { t } from '@vegaprotocol/i18n';
+import { Size } from '@vegaprotocol/datagrid';
+import * as Schema from '@vegaprotocol/types';
+import {
+  Dialog,
+  KeyValueTable,
+  KeyValueTableRow,
+  Tooltip,
+  VegaIcon,
+  VegaIconNames,
+  truncateMiddle,
+} from '@vegaprotocol/ui-toolkit';
+import type { Order } from '../order-data-provider';
+import { Link } from 'react-router-dom';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import { useCopyTimeout } from '@vegaprotocol/react-helpers';
+
+interface OrderViewDialogProps {
+  isOpen: boolean;
+  order: Order;
+  onChange: (open: boolean) => void;
+}
+
+export const OrderViewDialog = ({
+  isOpen,
+  order,
+  onChange,
+}: OrderViewDialogProps) => {
+  const [, setCopied] = useCopyTimeout();
+  return (
+    <Dialog open={isOpen} title={t('Order details')} onChange={onChange}>
+      <KeyValueTable>
+        <KeyValueTableRow key={'order-market'}>
+          <div data-testid={'order-market-label'}>{t('Market')}</div>
+          <div data-testid={`order-market-value`}>
+            <Link to={`/markets/${order.market?.id}`}>
```

Comment:
Use the `Links[Routes.MARKET](order.market.id)` from client-router.tsx

Reply:
updated